src/item.h

```diff
@@ -275,8 +275,7 @@ class item : public JsonSerializer, public JsonDeserializer
     /**
      * Whether the character needs both hands to wield this item.
      */
-    // TODO: make a reference. Make a const reference.
-    bool is_two_handed(player *u);
+    bool is_two_handed( const player &u ) const;
```

Comment:
Just pointing out, even better than a const reference would be passing in current strength as an integer.